assets/helpers/url.js

```diff
@@ -84,4 +100,5 @@ export {
   getQueryParams,
   addQueryParamToURL,
   getBaseDomain,
+  addQueryParamsToURL,
```

Comment:
There is already an `addQueryParamToURL` function, Is it possible to delete that function and only expose one function as a API?

Reply:
Yes, that's the plan. However, I'm a bit nervous to do that at the moment because that's used in several critical places. I'm going to test this new one out with a new feature I'm adding, and if that works fine across the browsers I'll go through and replace the old function everywhere.